src/services/mock/one/flow.cylc

```diff
@@ -100,7 +100,4 @@
             # reset stuff
             cylc unhold \
                 "${CYLC_SUITE_NAME}" "sleepy.${CYLC_TASK_CYCLE_POINT}"
-            cylc reset \
-                "${CYLC_SUITE_NAME}" "retrying.${CYLC_TASK_CYCLE_POINT}" \
-                -s 'succeeded'
```

Comment:
@oliver-sanders could you take a look at the `one` workflow, please? My Cylc-Fu is not good enough. I removed `cylc reset` after @hjoliver confirmed this subcommand was removed with SoD. But not sure how to re-write `one` now. Simply unhold'ing the task still resulted into a stalled workflow for me :cry: 

This is the last part missing, so that I can update the e2e tests, and then it should be ready for review :crossed_fingers: